cluster/cluster.yaml

```diff
@@ -1739,8 +1730,6 @@ Resources:
       VersioningConfiguration:
         Status: Suspended
       Tags:
-      - Key: application
-        Value: audittrail-adapter
```

Comment:
Same here, `component` tag instead.

Reply:
Done. It already had a tag `component` with value `resend`, I just changed it to `audittrail-adapter`.